examples/maxswerve/tests/test_swerveutils.py

```diff
@@ -0,0 +1,8 @@
+import swerveutils
+
+def test_stepTowardsCircular1():
+    current = 0.6408134451373411
+    stepsize = 0.3455804605358387
+    target = 0.0  # stepping towards zero direction
+    result = swerveutils.stepTowardsCircular(current=current, stepsize=stepsize, target=target)
+    assert abs(result) < abs(current), f"swerveutils: why stepping from {current} to {target} results in {result}?"
```

Comment:
pytest will show the values in the assert condition on failure iff there's no assert message.
```suggestion
    assert abs(result) < abs(current)
```

Reply:
@auscompgeek : done in 7b4483f3698e7c1b99faf56858915d80efd0e34b